src/modules/endpoint/DataContainer.js

```diff
@@ -0,0 +1,238 @@
+// @flow
+
+import {
+  CategoriesMapModel,
+  CityModel, EventModel,
+  LanguageModel
+} from '@integreat-app/integreat-api-client'
+import DatabaseContext from './DatabaseContext'
+import type { ResourceCacheStateType } from '../app/StateType'
+import DatabaseConnector from './DatabaseConnector'
+
+interface DataContainerInterface {
```

Comment:
@Schedulaar Should we `export default` the interface instead of the implementation. The interface is the thing that will be used almost everywhere in the code and I think one can easily make the mistake and accidentally import the implementation the way it is now.
I personally like `DataContainer` better as a name for the interface and call the implementation something like `DefaultDataContainer` or something similar.

Reply:
I have no preference here :thinking:

Reply:
I implemented it as suggested.